tests/python/data/forbidden_imports_sample.py

```diff
@@ -0,0 +1,10 @@
+from datetime import date
+from datetime import datetime
+
+import pandas as pd
```

Comment:
Is pandas definitely needed to do these tests?  That's a heavy package to add just for testing...  And any testing packages you need will probably need to be added with `poetry add --group dev <pkg>`.

Reply:
oh man you're right. i will remove. i'll show another alias..the temptation was too great.